src/cockpit/packages.py

```diff
@@ -195,6 +226,18 @@ def get_content_security_policy(self, origin):
     def serve_file(self, path, channel):
         filename = self.negotiate_file(path, channel.headers)
 
+        # HACK: if a translation file is missing, just return empty
+        # content. This saves a whole lot of 404s in the developer
+        # console when trying to fetch po.js for English, for example.
+        if filename is None and path.endswith('po.js'):
+            headers = {
+                'Access-Control-Allow-Origin': channel.origin,
+                'Cache-Control': 'no-cache, no-store',
```

Comment:
Not a huge fan of the duplication here.

Reply:
In general I feel like it would be better to handle this case by using the same serving code as you use below, but figuring out a way to serve the empty string instead of the contents of a file.  Something like the moral equivalent of `filename = '/dev/null'`, but without actually reading the file.

Reply:
I did a thing to reduce duplucation.